dockstore-webservice/src/test/java/io/swagger/api/impl/ToolsImplCommonTest.java

```diff
@@ -280,10 +282,15 @@ private void convertDockstoreWorkflowToTool(String toolname) {
         expectedToolVersion1.setMetaVersion(null);
         expectedToolVersion1.setVerified(true);
         expectedToolVersion1.setVerifiedSource("potatoTesterSource");
+        expectedToolVersion1.setImageName("");
+        expectedToolVersion1.setRegistryUrl("");
         json = gson.toJson(expectedToolVersion1);
         ToolVersion expectedToolVersion2 = gson.fromJson(json, ToolVersion.class);
         expectedToolVersion2.setName(REFERENCE3);
         expectedToolVersion2.setVerifiedSource("chickenTesterSource");
+        expectedToolVersion2.setImageName("");
+        expectedToolVersion2.setRegistryUrl("");
```

Comment:
Are these two lines needed?

Reply:
Yes, they were needed to pass the test. 
I think the gson handles nulls a bit differently from jackson which we're using elsewhere.